source/operations/troubleshooting.rst

```diff
@@ -164,11 +164,11 @@ For more details about encrypting or decrypting such files, see :ref:`Encrypting
 Logs
 ----
 
-Use the subcommands for ``mc support logs`` to :mc:`~mc support logs enable` or :mc:`~mc support logs disable` the submission of MinIO logs to SUBNET.
-You can also use :mc:`mc support logs status` to check if a log submission is in progress.
+Use the subcommands for ``mc support callhome`` to :mc:`~mc support callhome enable` or :mc:`~mc support callhome disable` the submission of MinIO logs to SUBNET.
+You can also use :mc:`mc support callhome status` to check if a log submission is in progress.
 
-Use :mc:`mc support logs show` command to display logs from the command line.
-Use the parameter flags for the :mc:`mc support logs show` command to limit the displayed logs by type or quantity.
+Use :mc:`mc admin logs` command to display logs from the command line.
+Use the parameter flags for the :mc:`mc admin logs` command to limit the displayed logs by type or quantity.
```

Comment:
Feels odd to lead with this without stating the flags. I'd drop it, or maybe say something like "The logs command supports filters for further limiting log output."

Reply:
👍